responsibleai/tests/test_feature_metadata.py

```diff
@@ -32,7 +32,7 @@ def test_feature_metadata_with_identity_feature(self):
         assert feature_metadata.dropped_features is None
         with pytest.raises(
                 UserConfigValidationException,
-                match='Identity feature name id given is not present'
+                match='The given identity feature name id, is not present'
```

Comment:
nit: remove the "," in the original error message and in the check here, eg:
```
The given identity feature name id is not present in user features.
```

Reply:
done.